testsuite/cluster-test/src/cluster_swarm/cluster_swarm_kube.rs

```diff
@@ -734,33 +729,23 @@ impl ClusterSwarmKube {
                         .as_ref()
                         .unwrap_or(&"".to_string()),
                     seed_peer_ip = fullnode_config.seed_peer_ip,
-                );
-                Some(serde_yaml::to_vec::<NodeConfig>(
-                    &NodeConfig::parse(&config)
-                        .map_err(|e| format_err!("Failed to parse config template : {}", e))?,
-                )?)
+                )
             }
             LSR(lsr_config) => {
-                let config = format!(
+                format!(
                     include_str!("configs/safetyrules.yaml"),
                     vault_addr = lsr_config.vault_addr.as_ref().unwrap_or(&"".to_string()),
                     vault_ns = lsr_config
                         .vault_namespace
                         .as_ref()
                         .unwrap_or(&"".to_string()),
-                );
-                Some(serde_yaml::to_vec::<SafetyRulesConfig>(
-                    &SafetyRulesConfig::parse(&config)
-                        .map_err(|e| format_err!("Failed to parse config template : {}", e))?,
-                )?)
+                )
             }
-            _ => None,
+            _ => "".to_string(),
         };
 
-        if let Some(node_config) = node_config {
-            self.put_file(node, pod_name, "/opt/libra/etc/node.yaml", node_config)
-                .await?;
-        }
+        self.put_file(node, pod_name, "/opt/libra/etc/node.yaml", Vec::from(node_config))
```

Comment:
Since we are here, can we refactor put_file little bit?

- Make it take `&[u8]` instead of String as last argument
- Here and in similar places you can use `.as_bytes()`
- In `put_file`, instead of `content.clone()`, use `content.to_vec()`

This will avoid redundant cloning of config